CHANGELOG.md

```diff
@@ -2,6 +2,10 @@
 
 ## Unreleased
 
+## 🧰 Bug fixes 🧰
+
+- `statsdreceiver`: Fix counter metric being dropped by Prometheus exporter (#7156)
```

Comment:
Why `statsdreceiver`? Looks like the change is made on prometheus exporter

Reply:
nice catch, sorry for this one, I'm waiting to finalize the solution from the above discussion with @jmacd @Aneurysm9 so will update this later as well.